src/common/AllExperiments/Experiments_Live.cs

```diff
@@ -0,0 +1,126 @@
+    using Microsoft.PowerToys.Telemetry;
+    using Microsoft.VariantAssignment.Client;
+    using Microsoft.VariantAssignment.Contract;
+    using Windows.System.Profile;
+
+#pragma warning disable SA1649 // File name should match first type name. Suppressed because it needs to be the same class name as Experiments_Inert.cs
+    public class Experiments
+#pragma warning restore SA1649 // File name should match first type name
+    {
+        public bool EnableLandingPageExperiment()
+        {
+            Experiments varServ = new Experiments();
+            varServ.VariantAssignmentProvider_Initialize();
+            var landingPageExperiment = varServ.IsExperiment;
+
+            return landingPageExperiment;
+        }
+
+        private void VariantAssignmentProvider_Initialize()
+        {
+            IsExperiment = false;
+
+            var vaSettings = new VariantAssignmentClientSettings
+            {
+                Endpoint = new Uri("https://default.exp-tas.com/exptas77/a7a397e7-6fbe-4f21-a4e9-3f542e4b000e-exppowertoys/api/v1/tas"),
+                EnableCaching = true,
+                ResponseCacheTime = TimeSpan.FromMinutes(5),
+            };
+
+            var vaClient = vaSettings.GetTreatmentAssignmentServiceClient();
+            var vaRequest = GetVariantAssignmentRequest();
+
+            try
+            {
+                var task = vaClient.GetVariantAssignmentsAsync(vaRequest);
+                TimeSpan ts = TimeSpan.FromMilliseconds(500);
+                if (task.Wait(ts))
+                {
+                    var result = task.Result;
+                    var allFeatureFlags = result.GetFeatureVariables();
+
+                    var assignmentContext = result.GetAssignmentContext();
+                    var featureNameSpace = allFeatureFlags[0].KeySegments[0];
+                    var featureFlag = allFeatureFlags[0].KeySegments[1];
+                    var featureFlagValue = allFeatureFlags[0].GetStringValue();
+
+                    if (featureFlagValue == "alternate")
+                    {
+                        IsExperiment = true;
+                    }
+
+                    PowerToysTelemetry.Log.WriteEvent(new OobeVariantAssignmentEvent() { AssignmentContext = assignmentContext, ClientID = AssignmentUnit });
+                }
+            }
+            catch (Exception ex)
+            {
+                Logger.LogError("Error getting variant assignments for experiment", ex);
+            }
+        }
+
+        public bool IsExperiment { get; set; }
+
+        private string? AssignmentUnit { get; set; }
+
+        private IVariantAssignmentRequest GetVariantAssignmentRequest()
+        {
+            string? clientID = string.Empty;
+
+            string workingDirectory = Environment.CurrentDirectory;
+            var exeDir = Directory.GetParent(workingDirectory)?.Parent?.Parent?.FullName;
+            string settingsPath = @"src\common\AllExperiments\settings.json";
```

Comment:
Can you help me understand where this is being saved? It looks like it's trying to save on the PowerToys installation folder.

Reply:
Does it make sense to add it to BugReport tool?

Reply:
It should be automatically there as long as it's in the right folder (%localappdata%\Microsoft\PowerToys). Just not sure what's going on here. at this point. @chenss3

Reply:
I went with your suggestion of making my own .json file for experimentation. I'm guessing I did not put it in the right spot - what directory should it be in, and what should it be called?

Reply:
Alongside the other settings files in "%localappdata%\Microsoft\PowerToys" and a good name might be "experimentation.json"

Reply:
Is this client id user or computer specific? Do we get problems having a json for each user profile?

Reply:
I made a modification and added a experimentation.json file to the user's Appdata/Local directory. Experimentation.json file is what is storing the clientid. Does this work? @jaimecbernardo